plugins/modules/aci_access_spine_interface_selector.py

```diff
@@ -0,0 +1,324 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Gaspard Micol (@gmicol) <gmicol@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "certified"}
+
+DOCUMENTATION = r"""
+---
+module: aci_access_spine_interface_selector
+short_description: Manage Fabric Access Policy Spine Interface Port Selectors (infra:SHPortS and infra:RsSpAccGrp)
```

Comment:
Do we also include the relationship classes in the short description? Thought we only did the main class? Not saying it is bad, but deviates from what we have been doing, should we change this everywhere?

Reply:
I let it on the short description on purpose to see our team's reaction. I think it's not needed. What we've been doing so far is fine for short description.
I'll remove it.